nodes.adoc

```diff
@@ -0,0 +1,1163 @@
+----
+
+
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create cronjob pi --image=perl --schedule='*/1 * * * *' -- perl -Mbignum=bpi -wle 'print bpi(2000)'
+----
+
+## Running background tasks on nodes automatically with daemon sets
+
+[source,terminal]
+----
+$ oc patch namespace myproject -p \
+    '{"metadata": {"annotations": {"openshift.io/node-selector": ""}}}'
+----
+
+
+[source,terminal]
+----
+$ oc create -f daemonset.yaml
+----
+
+## Freeing node resources using garbage collection
+
+[source,terminal]
+----
+$ oc label machineconfigpool worker custom-kubelet=small-pods
+----
+
+
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f gc-container.yaml
+----
+
+## Managing the maximum number of pods per node
+
+[source,terminal]
+----
+$ oc label machineconfigpool worker custom-kubelet=small-pods
+----
+
+## Managing nodes
+
+[source,terminal]
+----
+$ oc label machineconfigpool worker custom-kubelet=enabled
+----
+
+
+[source,terminal]
+----
+$ oc create -f <file-name>
+----
+
+
+[source,terminal]
+----
+$ oc create -f master-kube-config.yaml
+----
+
+## Modifying existing nodes in your {product-title} cluster
+
+[source,terminal]
+----
+$ oc adm cordon <node>
+----
+
+
+[source,terminal]
+----
+$ oc adm cordon node1.example.com
+----
+
+
+[source,terminal]
+----
+$ oc adm uncordon <node1>
+----
+
+## Allocating resources for nodes in an {product-title} cluster
+
+[source,terminal]
+----
+$ oc label machineconfigpool worker custom-kubelet=small-pods
+----
+
+
+[source,terminal]
+----
+$ oc label machineconfigpool worker custom-kubelet=small-pods
+----
+
+## Allocating specific CPUs for nodes in a cluster
+
+[source,terminal]
+----
+$ oc create -f <file_name>.yaml
+----
+
+## Viewing and listing the nodes in your {product-title} cluster
+
+[source,terminal]
+----
+$ oc adm top nodes
+----
+
+
+[source,terminal]
+----
+$ oc adm top node --selector=''
+----
+
+## Working with nodes
+
+[source,terminal]
+----
+$ oc adm cordon <node1>
+----
+
+
+[source,terminal]
+----
+$ oc adm drain <node1> <node2> [--pod-selector=<pod_selector>]
+----
+
+
+[source,terminal]
+----
+$ oc adm drain <node1> <node2> --force=true
+----
+
+
+[source,terminal]
+----
+$ oc adm drain <node1> <node2> --grace-period=-1
+----
+
+
+[source,terminal]
+----
+$ oc adm drain <node1> <node2> --ignore-daemonsets=true
+----
+
+
+[source,terminal]
+----
+$ oc adm drain <node1> <node2> --timeout=5s
+----
+
+
+[source,terminal]
+----
+$ oc adm drain <node1> <node2> --delete-local-data=true
+----
+
+
+[source,terminal]
+----
+$ oc adm drain <node1> <node2>  --dry-run=true
+----
+
+
+[source,terminal]
+----
+$ oc adm uncordon <node1>
+----
+
+
+[source,terminal]
+----
+$ oc label node <node> <key_1>=<value_1> ... <key_n>=<value_n>
+----
+
+
+[source,terminal]
+----
+$ oc label nodes webconsole-7f7f6 unhealthy=true
```

Comment:
Requires a declarative alternative

Reply:
@siamaksade This and the previous command are in the same bullet. I added on declarative for both,